src/cli/onefuzz/api.py

```diff
@@ -639,40 +639,40 @@ def create_with_config(self, config: models.TaskConfig) -> models.Task:
 
     def create(
         self,
-        job_id: UUID_EXPANSION,
-        task_type: enums.TaskType,
-        target_exe: str,
         containers: List[Tuple[enums.ContainerType, primitives.Container]],
+        job_id: UUID_EXPANSION,        
+        target_exe: str,
+        task_type: enums.TaskType,
         *,
```

Comment:
You need to not sort anything prior to `*,`.  Those are ordered params.  Everything post `*,` are named params and therefore can be sorted.